### ISSUE TITLE
[Android] not working with react-native 0.62.0

### ISSUE THREAD

**Issue by cuongnm2301** [magicismight/react-native-root-siblings#62]
It work normally in 0.61.5, but when I update to 0.62.0.
It work in ios only. pls check.
Many thanks

**Comment by crystalxmumu** [comment]
yes, the wrong is : Animated: 'useNativeDriver' was not specified. This is required option adn must be explicitly set to 'true' or 'false'

**Comment by charway** [comment]
have the same problem, hope to solve it ASAP.

**Comment by cuongnm2301** [comment]
> yes, the wrong is : Animated: 'useNativeDriver' was not specified. This is required option adn must be explicitly set to 'true' or 'false'

How can I fix this

**Comment by sunnylqm** [comment]
This library has nothing to do with Animated. So you may be using some other libraries that depend on this one. Check that first.

**Comment by cuongnm2301** [comment]
> This library has nothing to do with Animated. So you may be using some other libraries that depend on this one. Check that first.

I use only this one, try to run example but still not working on android

**Comment by tharyckgusmao** [comment]
Yes, this library does not use animation, but it does not work on android, IOS working.

**Comment by sunnylqm** [comment]
I can reproduce this issue and is looking into it

**Comment by sunnylqm** [comment]
A workaround is to inject a RootSiblingParent in your app root node like this https://github.com/magicismight/react-native-root-siblings/issues/52#issuecomment-563252331
But `destroy()` still not work.
Related to https://github.com/facebook/react-native/blob/0b9ea60b4fee8cacc36e7160e31b91fc114dbc0d/Libraries/ReactNative/AppRegistry.js#L298-L309  @magicismight

**Comment by Foveluy** [comment]
very funny is that wrapping a `<RootSiblingParent/>` to the very root of my app, this lib is work again in RN-0.6.2-Android.

so the code looks like:

```js
export default () => {
  // fixer https://github.com/magicismight/react-native-root-siblings/issues/62
  const Wrapper = Platform.OS === 'ios' ? React.Fragment : RootSiblingParent;
  return (
    <Wrapper>
      <App />
    </Wrapper>
  );
};
```

**Comment by cuongnm2301** [comment]
tks @Foveluy it work for me too

**Comment by wslbxx123** [comment]
I still cannot use destroy function when wrapping a <RootSiblingParent/>

**Comment by cybercoder** [comment]
Still not works when call ```Toast.show```

**Comment by wuxiii** [comment]
> function

same to me

**Comment by lybrooks** [comment]
same  to me

**Comment by sunnylqm** [comment]
There is a workaround above. Please read it before you post "+1"/"same", which is meaningless.

**Comment by StagasaurusRex** [comment]
With the workaround `destroy()` does not work correct? Is there a plan to update this library to support 0.62 & 0.63 without the workaround? Or at least a way for `destroy()` to work?

**Comment by keepzouba** [comment]
Use version 3.2.3
